### DISCUSSION TITLE
Why can't HxSelect bind to a nullable int?

### DISCUSSION THREAD

**Discussion by franky1017** [havit/Havit.Blazor#594] [Q&A] [answered]
When using HxSelect, I am not able to bind to a nullable int property. I mean the control itself has Nullable attribute and can be set to true so why can't control bind to a nullable int?

**Comment by hakenr** [comment]
We are using HxSelect with nullable properties on daily basis. What is the symptom of not being able to bind to nullable property?

E.g. most of the "pickers" here (those based on `HxSelectBase`) are used with `Nullable<int>` properties: https://github.com/mensagymnazium/IntranetGen3/tree/master/Web.Client/Components/Pickers

**Comment by franky1017** [comment, marked as answer]
Oh, I got it! I need to make the TValue to int? first. Thanks for helping :-)
